openmetadata-service/src/main/java/org/openmetadata/service/secrets/SecretsManagerUpdateService.java

```diff
@@ -103,14 +102,6 @@ private void updateIngestionPipelines() {
     retrieveIngestionPipelines().forEach(this::updateIngestionPipeline);
   }
 
-  private void updateWorkflows() {
```

Comment:
@pmbrull is this only used for automation_workflow  for test connection? just thinking from the persepective if there are schema changes relevant to worflow , and we have encrypted something it will to be run from this flow to have correct update new schema which is encrypted?

Reply:
automation_workflow should always be empty. it's only used to pass info between server (UI) <> client. No data should be kept there so we can remove the SM updates